### PR TITLE
add sensitivity analysis tool for layer-wise FIT and Hessian trace

### PR DESCRIPTION
**Pull request by Hanxian97**
Summary:
This is a sensitivity analysis tool to obtain the layer-wise average Hessian trace and FIT (Fisher Information Trace). (Hanxian's intern project milestone2)

Test Plan:
python scripts/FIT.py --checkpoint=MODEL_CHECKPOINT_PATH
python scripts/Hessian_vhp.py --checkpoint=MODEL_CHECKPOINT_PATH--layer_id=DESIRED_LAYER_TO_COMPUTE
python scripts/Hessian_grad.py --checkpoint=MODEL_CHECKPOINT_PATH --layer_id=DESIRED_LAYER_TO_COMPUTE

### CODE REVIEW

File: torchao/quantization/prototype/mixed_precision/scripts/FIT.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import torch
```
Comment (andrewor14) [inline torchao/quantization/prototype/mixed_precision/scripts/FIT.py:1]: nit: can you use lowercase names for these 3 files?

**Review by andrewor14 — APPROVED** [review]
Thanks! In the future we should add a short README to explain how to use this script. We can do that in a separate PR

File: torchao/quantization/prototype/mixed_precision/scripts/FIT.py (hunk 1 above)
Comment (Hanxian97) [reply torchao/quantization/prototype/mixed_precision/scripts/FIT.py]: I have renamed them

**Comment by msaroufim** [comment]
@Hanxian97 @andrewor14 is this ready to be merged?

**Comment by andrewor14** [comment]
Merging this. Thanks!

**Pull request merged**